src/main/java/com/uwetrottmann/tmdb2/entities/TaggedImage.java

```diff
@@ -3,8 +3,11 @@
 public class TaggedImage extends Image {
 
     public String id;
-    public String image_type;
     public String media_type;
+    public String title;
```

Comment:
Sorry, but that does not look right. Looking at [the docs](https://developers.themoviedb.org/3/people/get-tagged-images) and the API response for `PeopleServiceTest#test_tagged_images` and `PeopleServiceTest#test_summary_with_append_to_response` there is no title or overview (only inside the media object).

Reply:
Now that's quite odd. Because i swear i was just looking up on the docs for updates for all the entities, and it was there. Well that's odd.... Because it passed all the tests when i pushed it. i'll be checking it out